e2e-product-testing/cypress/integration/Project/on-project-page.ts

```diff
@@ -0,0 +1,7 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+Given(`I am on Projects page`, () => {
+  cy.visit(`https://marxan.vercel.app/projects`, {
```

Comment:
I suggest to generate these URLs via a URL factory that reads the base URL from an env var and appends paths according to which part of the app we are testing (or at least allows to set the base URL via env var).
Can be in a separate PR, here we're dealing with the initial PoC so we can merge with hardcoded URLs if we wanted.

Reply:
I agree on separate PR - as per requirements we want the minimal setup to work with ;)